analysis/Scripts/tuples_prep_main.py

```diff
@@ -1,58 +0,0 @@
-import ROOT, os, sys, Imports
-from ROOT import TChain, TFile
-sys.path.append('./tuples_prep_scripts/') #step necessary to import the scripts placed in the subdirectory of the current directory
-import Strip, SetBranch, SplitScript
-
-folders_dict = Imports.getFoldersDict()  #a dictionary containing the details of the all the years' data according to joblog.txt
-
-dir = "/dcache/bfys/scalo/"
-
-particle = "Lc"
-
-for element in folders_dict:
-    #These commented lines should be uncommented in case nTracks is a variable needed for the PID efficiency calculation
-    if int(element) < 47:
-       extra_variables = ["lcplus_Hlt1TrackAllL0Decision_TOS", "lcplus_Hlt2CharmHadD2HHHDecision_TOS"]
-       run = 1
-    else:
-       extra_variables = ["nTracks", "lcplus_Hlt1TrackMVADecision_TOS"]
-       run = 2
```

Comment:
maybe archive these instead of deleting?

Reply:
As in my answers above, I can do an archive folder for the Scripts directory